code/game/jobs/job/medical.dm

```diff
@@ -45,6 +45,7 @@
 		H.put_in_hand(GRASP_RIGHT_HAND, new H.species.survival_gear(H))
 	else
 		H.equip_or_collect(new H.species.survival_gear(H.back), slot_in_backpack)
+	H.mind.store_memory("Frequencies list: <br/><b>Command:</b> [COMM_FREQ] <b>Medical:</b> [MED_FREQ]")
```

Comment:
God I wish it was defined as CMD instead of `COMM_FREQ` I keep reading "Communication Frequency"